test/specs/scenarios/crossDeviceDocumentVideoCapture.js

```diff
@@ -0,0 +1,273 @@
+          copy
+        )
+        await takePercySnapshotWithoutOverlay(
+          driver,
+          `User is asked what type of license they have ${lang}`
+        )
+        documentVideoCapture.paperDocumentButton().click()
+        documentVideoCapture.frenchPaperDrivingLicenseOverlay().isDisplayed()
+        documentVideoCapture.userIsGivenInstructionsForProfilePhotoSide(copy)
+        await takePercySnapshotWithoutOverlay(
+          driver,
+          `User is told to lay the document flat ${lang}`
+        )
+        documentVideoCapture.userStartsAndCompletesFirstPartOfCapture(copy)
+        documentVideoCapture.overlayPlaceholder().isDisplayed()
+        documentVideoCapture.progressSteps().isDisplayed()
+        documentVideoCapture.userIsToldToTurnDocumentToShowTheOuterPages(copy)
+        documentVideoCapture.userIsToldToKeepDocumentInFullViewAtAllTimes(copy)
+        await takePercySnapshotWithoutOverlay(
+          driver,
+          `User is shown step 2 of 2 for drivers license flow and asked to slowly turn the document around ${lang}`
+        )
+        userFinishesRecordingAndIsTakenToConfirmationScreen(copy)
+        documentVideoConfirm.uploadAndWaitForSpinner(copy)
+      })
+
+      it('should start the ANSII flow for Residence permit flow and attempts to upload @percy', async () => {
+        userStartsCrossDeviceFlowForUKResidentPermit()
+        documentVideoCapture.cardOverlay().isDisplayed()
+        documentVideoCapture.userIsToldToKeepTheFrontSideOfTheDocumentWithinTheFrame(
+          copy
+        )
+        await takePercySnapshotWithoutOverlay(
+          driver,
+          `User is told to keep the front side within the frame ${lang}`
+        )
+        documentVideoCapture.userStartsAndCompletesFirstPartOfCapture(copy)
+        documentVideoCapture.progressSteps().isDisplayed()
+        documentVideoCapture.userIsToldToSlowlyTurnDocumentToShowTheBack(copy)
+        documentVideoCapture.userIsToldToKeepDocumentInFullViewAtAllTimes(copy)
+        await takePercySnapshotWithoutOverlay(
+          driver,
+          `User is asked to slowly turn the document around to show the back ${lang}`
+        )
+        userFinishesRecordingAndIsTakenToConfirmationScreen(copy)
+        documentVideoConfirm.uploadAndWaitForSpinner(copy)
+      })
+
+      it('should show Camera not working error in ANSII flow @percy', async () => {
+        userStartsCrossDeviceFlowForItalianIdCard(copy)
+        documentVideoCapture.overlayPlaceholder().isDisplayed()
+        documentVideoCapture.userIsShownCameraNotWorkingError(copy)
+        await takePercySnapshotWithoutOverlay(
+          driver,
+          `User is shown Camera not working error ${lang}`
+        )
+        documentVideoCapture.userCanDismissError()
+        documentVideoCapture.cameraNotWorkingErrorIsNotSeen()
+      })
+
+      it('should show Looks like you took too long error in ANSII flow @percy', async () => {
```

Comment:
I think (but not 100% sure) the acronym is ANSSI
```suggestion
      it('should show "Looks like you took too long" error in ANSSI flow @percy', async () => {
```